README.md

```diff
@@ -1 +1,210 @@
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.google.com/chrome/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[A .html file that includes a header and main section.]**
+- **[A .css file that includes styles for the whole project]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+ NOT AVAILABLE AT THE MOMENT
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+* GitHub account
+
+* Visual Studio Code
+
+* Git
+
+* Figma
+
+### Setup
+
+Clone this repository to your desired folder:
+
+-git clone https://github.com/Chimane1987/Mobile-portfolio.git my-folder
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Install this project with:
+
+* npm init -y
+
+* npm install --save-dev hint@7.x
+
+* npx hint .
+
+* Stylelint
+
+* npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project, execute the following command:
+
+* Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
+
+* The entire codebase looks like it was written by one person.
+
+* Programmers can focus on solving problems instead of cleaning up the code.
+### Run tests
+
+To run tests, run the following command:
+
+* npx stylelint "**/*.{css,scss}"
+### Deployment
+
+You can deploy this project using:
+
+GitHub
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Author1**
+
+- GitHub: [https://github.com/Chimane1987]
+
+- Twitter: [https://twitter.com/Cheeks22670006]
+
+- Linkedin: [https://www.linkedin.com/in/elias-molefe-2a83aa18a/]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **[index.html]**
+- [ ] **[style.css]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/issues](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project please give it a star 🌟 and follow me on GitHub 🙏.
+
+
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank Microverse for the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Chimane1987/Hello-Project/blob/main/LICENSE) licensed.
```

Comment:
[optional] Please kindly fix this broken link for us to have a professional repo. 👍